src/VisualStudio/CSharp/Impl/Options/CSharpVisualStudioOptionStorageReadFallbacks.cs

```diff
@@ -57,16 +57,25 @@ public NewLinesForBraces()
         }
 
         public Optional<object?> TryRead(string? language, TryReadValueDelegate readValue)
-            => TryReadFlags(s_storages, readValue, out var intValue) ? (NewLineBeforeOpenBracePlacement)intValue : default(Optional<object?>);
+            => TryReadFlags(s_storages, (int)CSharpFormattingOptions2.NewLineBeforeOpenBrace.DefaultValue, readValue, out var intValue) ? (NewLineBeforeOpenBracePlacement)intValue : default(Optional<object?>);
     }
 
-    private static bool TryReadFlags(ImmutableArray<(string key, int flag)> storages, TryReadValueDelegate read, out int result)
+    private static readonly object s_true = true;
+    private static readonly object s_false = false;
```

Comment:
💡 These are already defined as `Boxes.True` and `Boxes.False`.

Reply:
Thanks, I knew we must have these but couldn't find them.